### PR TITLE
Added Gavin Reid advice page

### PR DESCRIPTION
**Pull request by gjlreid**
New Advice page added along with link to the list of advice page

### CODE REVIEW

**Comment by twosavoie** [comment]
Hi @gjlreid,

This is great!

There's one small change to make. In your advice file, there's a link to a **Gavin Reid.css** file. But this file isn't in your pull request. Can you either add the Gavin Reid.css file to the css folder or change the link back to the main.css file. Either will work so it's totally up to you!

If you decide to use your own css file, be sure to add a dash between "Gavin" and "Reid". Computers find spaces in file names confusing. 🙂 

Once you've made that change, please add the change with `git add -A`, commit as per usual and push up your advice branch once more, which will automatically update this pull request.

Then we should be all set to merge in your code. Let us know if you have any questions!

Best,
Lisa

**Comment by twosavoie** [comment]
Hi @gjlreid,

Excellent question. You'll want to push the advice branch instead of the main branch. Before you do, check to make sure the changes were made on the advice branch. 

Best,
Lisa

**Comment by twosavoie** [comment]
Hi @gjlreid,

Just checking to see if you need any help with this pull request. Let us know if you have any questions!

Best,
Lisa

File: advice/Gavin-Reid-advice.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html>
+
+    <head>
+        <meta charset="utf-8">
+        <link rel="stylesheet" type="text/css" href="../css/Gavin Reid.css">
```
Comment (twosavoie) [inline advice/Gavin-Reid-advice.html:6]: Hi @gjlreid,

You're super close! You added a link to your own CSS file in the <head> of your advice.html file. But the css file you linked to isn't in the css folder. If you've decided not to add one, you could change it back to the main.css file. Either way would be completely fine!

Best,
Lisa

**Comment by gjlreid** [comment]
Hi Lisa,

I'm really confused. I did all that. I added a link to my own CSS file in
html and added the file to the css folder. I then pushed the advice
changes, this was 10 days ago. I now can't see any of my work on GitHub.

Do I need to do it all again? Where is the best place to start?
Thanks,
Gavin

**Comment by twosavoie** [comment]
Hey @gjlreid,

We haven't heard from you in a while, so we're going to go ahead and close this pull request. Feel free to submit a new one when you're ready!

Best,
Lisa

**Pull request closed without merging**